src/models/models.jl

```diff
@@ -0,0 +1,185 @@
+export PolarForm, get, bounds, powerflow
+export State, Control, Parameters, NumberOfState, NumberOfControl
+
```

Comment:
Also: I welcome any feedback on the naming. I am not sure I am explicit enough there :-)

Reply:
I think you are being explicit enough.